time timeStamp
int32 xPosition
int32 yPosition
int32 zPosition
int32 xPredicted
float32 velocityZ 
